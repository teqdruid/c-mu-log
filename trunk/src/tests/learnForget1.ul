stack("s1");
stack("s2");
stack("s3");

f() {
    @print("Removing: ", $s);
    @forget( stack($s); );

     $s == "s1";
}

l() {
    @print("Learning: ", $s);
    @learn( stack($s); );

    $s == 6;
}

main() {OR:
    @print($s);
    f();
    l();
    stack($s);
}      
