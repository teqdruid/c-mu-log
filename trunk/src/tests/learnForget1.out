Removing: 's1'
Any

 ^^^  Solution  ^^^ 

Learning: 6
Any

 ^^^  Solution  ^^^ 

6

 ^^^  Solution  ^^^ 

's2'

 ^^^  Solution  ^^^ 

's3'

 ^^^  Solution  ^^^ 

No more solutions
